graphql/schema.graphql

```diff
@@ -506,6 +506,11 @@ type FileDiff {
   description: String!
 }
 
+type TriggerAlias {
```

Comment:
Evergreen does have the concept of trigger aliases that aren't for patches, so I think to keep this consistent/clear this should still be named PatchTriggerAlias.